eng/BuildTask.targets

```
@@ -84,18 +84,24 @@
                Pack="false" />
   </ItemGroup>
 
-  <!-- Publish .NET Core assets and include them in the package under $(BuildTaskTargetFolder) directory. -->
+  <!-- Publish .NET Core assets and include them in the package under $(BuildTaskTargetTfmSpecificFolder) directory. -->
   <Target Name="_AddBuildOutputToPackageCore" DependsOnTargets="Publish" Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
+    <PropertyGroup>
+      <BuildTaskTargetTfmSpecificFolder Condition="'$(BuildTaskTargetTfmSpecificFolder)' == ''">$(BuildTaskTargetFolder)/($TargetFramework)</BuildTaskTargetTfmSpecificFolder>
```

We already have a property for the root level folder `BuildTaskTargetFolder`. Continue to respect that:

```suggestion
      <BuildTaskTargetTfmSpecificFolder Condition="'$(BuildTaskTargetTfmSpecificFolder)' == ''">($TargetFramework)</BuildTaskTargetTfmSpecificFolder>
```